Runtime/Scripts/MeshSyncServer.cs

```diff
@@ -506,6 +506,14 @@ public void StartServer()
     [SerializeField] private int  m_serverPort      = MeshSyncConstants.DEFAULT_SERVER_PORT;
 #if UNITY_EDITOR
     [SerializeField] bool m_foldServerSettings = true;
+
+    internal IDCCLauncher m_DCCInterop;
+    internal UnityEngine.Object m_DCCAsset;
+
+    public UnityEngine.Object DCCAsset
```

Comment:
Another possible use case that I have in mind, is that, it's possible that users are using both Blender files and other DCC files in their project. Let's say it's Maya in this case

If they are using mostly Maya files with MeshSync, and blend files are just there as assets, then linking blend files with MeshSyncServer will confuse them.

Reply:
As discusses in the call, it couuld be any kind of asset, if there is no handler for the file we show an error dialog.